<!--
	Simple Fetch wall stare functionality

	Ali Jones

	This assumes the Fetch simuation is already running. It launches two nodes
-->

<launch>

	<!-- Start "drive forward" node, moves the Fetch based on laser scan info-->
	<node name="drive_forward" pkg="rob599_hw1" type="drive_forward.py"/>

	<!-- Start node to trim the laser scan to only in front of the Fetch-->
	<node name="trim_msg" pkg="rob599_hw1" type="trim_laser_scan.py"/>

	<!-- Start node to fit a line to the wall using a marker -->
	<node name="wall_fit" pkg="rob599_hw1" type="wall_fit.py"/>

</launch>
